### ISSUE TITLE
Can not place headline

### ISSUE THREAD

**Issue by dojineko** [vuejs/vuepress#42]
hello. im trying VuePress.
i encountered a troubled problem and i will report it.

## Environment

- Node: 8.9.1
- VuePress: 0.3.1
     - .vuepress/config.js is not exist, all default settings

## Reproduction
Please put this markdown down.

```md
# Headline1
headline1 contains

## Headline2
headline2 contains

```

and, the error occurred like this. (TypeError: renderChildren(...) is not iterable)
when remove headline2 or replace to level1 headline, this error no occur.

```bash
$ npx vuepress build

Rendering static HTML...
Rendering page: /Error rendering /:
TypeError: renderChildren(...) is not iterable
    at children.map.c (server-bundle.js:2005:10)
    at Array.map (<anonymous>)
    at renderChildren (server-bundle.js:2001:61)
    at render (server-bundle.js:1978:21)
    at renderWithStyleInjection (server-bundle.js:195:16)
    at createFunctionalComponent (~/reproduction/node_modules/vue/dist/vue.runtime.common.js:4058:30)
    at createComponent (~/reproduction/node_modules/vue/dist/vue.runtime.common.js:4248:12)
    at _createElement (~/reproduction/node_modules/vue/dist/vue.runtime.common.js:4418:15)
    at createElement (~/reproduction/node_modules/vue/dist/vue.runtime.common.js:4355:10)
    at vm._c (~/reproduction/node_modules/vue/dist/vue.runtime.common.js:4487:42)
    at server-bundle.js:1944:521
    at VueComponent.renderList [as _l] (~/reproduction/node_modules/vue/dist/vue.runtime.common.js:3703:16)
    at VueComponent.Sidebarvue_type_template_id_5e11e29e_render (server-bundle.js:1944:258)
    at VueComponent.Vue._render (~/reproduction/node_modules/vue/dist/vue.runtime.common.js:4542:22)
    at renderComponentInner (~/reproduction/node_modules/vue-server-renderer/build.js:7521:25)
    at renderComponent (~/reproduction/node_modules/vue-server-renderer/build.js:7491:5)

Success! Generated static files in .vuepress/dist.
```

perhaps it might be related

- https://github.com/vuejs/vuepress/issues/34
- https://github.com/vuejs/vuepress/issues/19

thanks 😇

**Comment by ulivz** [comment]
Thanks, this issue has been fixed at: 42f63a8d710183c11dba7ba3764b28b09f511927, and new release hasn't released yet.

**Comment by dojineko** [comment]
very nice! I am looking forward to the release of it! 👍

**Comment by dojineko** [comment]
close this issue. when it would not be fixed after release `42f63a8`, i will reopen.
